<launch>
<!-- Pick Objects -->
<node name="add_markers_node" pkg="add_markers" type="add_markers" output="screen"/>
</launch>